--- googleplay.py_old
+++ googleplay.py
@@ -615,7 +615,7 @@
         params = {'ot': str(offerType),
                   'doc': packageName,
                   'vc': str(versionCode)}
-        self.log(packageName)
+#        self.log(packageName)
         response = requests.post(PURCHASE_URL, headers=headers,
                                  params=params, verify=ssl_verify,
                                  timeout=60,
