--- gplaycli.py_old
+++ gplaycli.py
@@ -252,11 +252,10 @@
 		for position, (detail, item) in enumerate(zip(details, pkg_todownload)):
 			packagename, filename = item
 
-			if filename is None:
-				if self.append_version:
-					filename = "%s-v.%s.apk" % (detail['docid'], detail['details']['appDetails']['versionString'])
-				else:
-					filename = "%s.apk" % detail['docid']
+			if self.append_version:
+				filename = "%s-v.%s.apk" % (detail['docid'], detail['details']['appDetails']['versionString'])
+			else:
+				filename = "%s.apk" % detail['docid']
 
 			logger.info("%s / %s %s", 1+position, len(pkg_todownload), packagename)
 
